src/order/rel_iso.lean

```diff
@@ -560,40 +579,33 @@ lemma order_iso.map_bot [order_bot α] [order_bot β]
   f ⊥ = ⊥ :=
 by { rw [eq_bot_iff, ← f.apply_symm_apply ⊥, ← f.map_rel_iff], apply bot_le, }
 
-lemma rel_iso.map_top [order_top α] [order_top β]
+lemma order_iso.map_top [order_top α] [order_top β]
```

Comment:
@awainverse I'm assuming these renamings were missed in #3838?

Reply:
Yeah, I think I had trouble making all of these namespace changes work for some reason I've forgotten. Since it all builds, this naming convention's definitely better.